reference/legacy-upgrade.md

```diff
@@ -2,7 +2,18 @@
 
 A legacy installation of Mayastor (1.0.5 and below) cannot be seamlessly upgraded and needs manual intervention.
 Follow the below steps if you wish to upgrade from Mayastor 1.0.x to Mayastor 2.1.0 and above.
-Mayastor uses etcd as a persistent datastore for its configuration. As a first step, take a snapshot of the etcd. The detailed steps for taking a snapshot can be found in the etcd [documentation](https://etcd.io/docs/v3.3/op-guide/recovery/).
+Mayastor uses etcd as a persistent datastore for its configuration. As a first step, take a snapshot of the etcd. To do so, execute:
+{% tab title="Commands" %}
+```text
+kubectl exec -it mayastor-etcd-0 -n mayastor -- bash
+kubectl cp -n mayastor mayastor-etcd-0:/tmp/snapshot.db ./snapshot.db
+ls -lrt snapshot.db
+kubectl exec -it mayastor-etcd-0 -n mayastor -- bash
```

Comment:
Are the steps repeated here?

Reply:
Do we need to take the snapshot on every pod of etcd? or only on the master pod?
cc @datacore-tilangovan

Reply:
Hi Vishnu, taking snapshot from one pod is enough.